src/commands/init.ts

```diff
@@ -91,7 +99,7 @@ export function action() {
       {
         type: "input",
         name: "speechPath",
-        message: "Specify folder path to save interaction model",
+        message: "Specify folder path to save interaction model and manifest",
```

Comment:
instead of `folder path` , better say `directory path`

https://stackoverflow.com/questions/5078676/what-is-the-difference-between-a-directory-and-a-folder